keyvault-argo.tf

```diff
@@ -0,0 +1,108 @@
+# This was throwing a duplicate resource name error, so I appended -extend to the alias
+data "azurerm_client_config" "current-extended" {}
+
+# This is the resource that I see in the portal, and referenced in the script.
+# I could not find it listed in this Terraform Repo. Probably can just run a terraform import to link the resource with this code block.
```

Comment:
Don't run the CLI `terraform import` command. Instead, add an import blocks right above all the resources that already exist. See https://developer.hashicorp.com/terraform/language/import

Reply:
Added import blocks for all relevant resources in this PR.